File: DCAnalyticsWebShell/ClientApp/src/models/FieldInspection.ts
import { Section } from "./Section";
import { Statuses } from "../enums/enums";

export class FieldInspection {
  public OID: number;
  public Key: string;
  public DateTime: Date;
  public CreatedBy: string
  public Status: Statuses;
  public FarmerKey: string;
  public FieldName: string;
  public Sections: Section[];
  public Coordinates: string[];
  ConfigurationId: string;
  Deleted : boolean
}

File: DCAnalyticsMobile/DCAnalyticsMobile/Models/FieldInspection.cs
using DCAnalytics;
using System;
using System.Collections.Generic;
using System.Text;

namespace DCAnalyticsMobile.Models
{
    public class FieldInspection: DataCollectionObject
    {
        public DateTime DateTime { get; set; }
        public Statuses Status { get; set; }
        public string FarmerKey { get; set; }
        public string Template { get; set; }
        public string FieldName { get; set; }
        public List<Section> Sections { get; set; }
        public List<string> Coordinates { get; set; }
        public int ConfigurationId { get; set; }
    }
}
